packages/web-runtime/src/container/bootstrap.ts

```diff
@@ -36,6 +36,37 @@ import { Resource } from '@ownclouders/web-client'
 import PQueue from 'p-queue'
 import { extractNodeId, extractStorageId } from '@ownclouders/web-client/src/helpers'
 
+const getEmbedConfigFromQuery = (
+  doesEmbedEnabledOptionExists: boolean
+): RawConfig['options']['embed'] => {
+  const config: RawConfig['options']['embed'] = {}
+
+  if (!doesEmbedEnabledOptionExists) {
```

Comment:
I'm not 100% sure that the protection for overwriting the config option is necessary but my thinking was - if anybody would actually set the embed mode in the config itself, it feels like they would not want it to be overwritten via the query param. And it would work both ways as I could either force the embed mode or disable it so that it cannot be embedded...

Reply:
My feeling is that we don't need the check. But fine either way. 👍